### DISCUSSION TITLE
swmaestro-coding-test/

### DISCUSSION THREAD

**Discussion by giscus** [koderpark/BlogComment#5] [Announcements]
# swmaestro-coding-test/

SWM(소마) 란?

과학기술정보통신부의 주도 아래 진행되는 국가과정으로, 창업에 관심이 있는 소프트웨어 분야 인재를 지원하는데에 그 초점이 맞추어진 교육과정입니다.
컨셉을 한 문장으로 정리해보자면

우리가 모든 준비를 해줄테니 넌 프로젝트만 만들어라!

에 제일 가깝습니다.

저기서의 "모든 준비" 에는

 * 월 장학금 100만원
 * IT기기 구입비 150만원
 * 프로젝트 개발 도중의 비용 지원

등등이 있습니다.

https://blog.koder.page/swmaestro-coding-test/

**Comment by SeHyun-Kim04** [comment]
소마 15기 어서오고 ^^
